tests/operator_test.go

```diff
@@ -748,6 +721,38 @@ var _ = Describe("ALL Operator tests", func() {
 				err = utils.WaitForDeploymentReplicasReady(f.K8sClient, f.CdiInstallNs, deploymentName)
 				Expect(err).ToNot(HaveOccurred())
 			})
+
+			It("CDI ready metric value as expected when ready to use", func() {
+				if !utils.IsOpenshift(f.K8sClient) {
```

Comment:
Same not sure if this openshift specific at this point. kubevirtci has a prometheus option now.